Automattic/wp-calypso
# composite-checkout: Add Alipay payment method

#### Changes proposed in this Pull Request

This adds the Alipay Stripe redirect payment method to composite checkout.

### Screenshots

Old checkout:

![old-alipay](https://user-images.githubusercontent.com/2036909/85627541-a9b92e80-b63c-11ea-8dd7-b00032586653.png)

New checkout closed:

![new-alipay-closed](https://user-images.githubusercontent.com/2036909/85627553-ade54c00-b63c-11ea-9670-8b6d553667db.png)

New checkout open:

![new-alipay-open](https://user-images.githubusercontent.com/2036909/85627567-b2116980-b63c-11ea-87f3-dd16499dc531.png)


#### Testing instructions

- Apply D45446-code to force Alipay to be available.
- Sandbox the store.
- Add a plan to your cart and visit checkout. 
- Add `?flags=composite-checkout-force` to the query string in the URL and reload the page because currently it is disabled for international currencies.
- Verify that "Alipay" is an option displayed in the final payment step.
- Verify that you cannot pay if the "Name" field is empty.
- Fill out the "Name" field and submit the payment.
- Verify that you are redirected to a Stripe test page.
- Click to accept the payment and verify that the payment completes successfully and that the plan is purchased.
